spec/lcm/load/user_bricks_load_spec.rb

```diff
@@ -85,18 +79,27 @@ def user_in_domain(user_name)
       @domain.create_users(user_array)
       users_csv = ConfigurationHelper.csv_from_hashes(user_array)
       Support::S3Helper.upload_file(users_csv, @test_context[:s3_key])
-      data_product_id = "testing-data-product-#{@suffix}"
-      @data_product = @domain.create_data_product(id: data_product_id)
+      @data_product_id = "testing-data-product-#{@suffix}"
+      @data_product = @domain.create_data_product(id: @data_product_id)
       @master_project = @rest_client.create_project(title: "Test MASTER project for #{@suffix}", auth_token: LcmConnectionHelper.environment[:prod_token])
       @segment = @data_product.create_segment(segment_id: "testing-segment-#{@suffix}", master_project: @master_project)
       @segment.create_client(id: 'testingclient', project: @project.uri)
       (1..project_count).each do |i|
-        proj = @rest_client.create_project(title: "Test MINOR project with testing_client_#{i} for #{@suffix}", auth_token: LcmConnectionHelper.environment[:prod_token])
-        @segment.create_client(id: "testing_client_#{i}", project: proj.uri)
-        project_array << proj
+        project_helper = Support::ProjectHelper.create(
+          client: @rest_client,
+          title: "Test MINOR project with testing_client_#{i} for #{@suffix}",
+          auth_token: LcmConnectionHelper.environment[:prod_token],
+          environment: 'TESTING'
```

Comment:
warning: Style/IndentHash: Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is. (rubocop)